### DISCUSSION TITLE
Is possible to stream from xbox dev mode to a client device?

### DISCUSSION THREAD

**Discussion by Gp-triple** [TheElixZammuto/moonlight-xbox#147] [Ideas]
Is possible to stream from xbox dev mode to a client device?
So the xbox dev mode is the server for stream emulator and other homebrew to any client device

Thanks

**Comment by GuiDev1994** [comment]
Moonlight is a client application, so it's not possible, you will need a Server Side like Sunshine or what the PS5 do to stream it.

**Reply by Gp-triple** [reply]
Understand, thanks
